templates/style/_navbar.scss

```diff
@@ -341,6 +341,11 @@ div.nav-container {
             animation: rotating_text 2s linear infinite;
         }
     }
+
+    #platforms {
+        max-height: 50vh;
```

Comment:
I'm not able to get horizontal scrollbar on either chrome or firefox. Which browser did you use?

Also: I didn't use `100vh` because it is extremely "buggy" on mobile: the viewport changes size depending you have the browser top navbar or not. Sometimes it includes it into the count, sometimes it doesn't. More information about it here: https://stackoverflow.com/questions/37112218/css3-100vh-not-constant-in-mobile-browser

I'll increase the navbar to `75vh` to make it taller though.

Reply:
I'm using firefox on windows

Reply:
Just to confirm, did you use `overflow-y` and not `overflow`?

Reply:
I didn't do anything, that's just what it looked like running this PR.

Reply:
Interesting, it might be because of the scrollbar (adding the one for y makes the width not enough anymore for x?). I added `width: fit-content`, can you check it please?

Reply:
To get the ellipsis you need to apply it on the element with the text

```css
#platforms li a {
  text-overflow: ellipsis;
  overflow-x: hidden;
}
```

Reply:
Good catch! Wrong copy/paste...